doc/nrf/libraries/networking/nrf_cloud_pgps.rst

```diff
@@ -90,7 +90,6 @@ There are three ways to define this storage location:
   By default, this partition is stored in the main SoC flash.
   This partition can optionally be located in external flash for the nRF9160 development kit version 0.14.0 and later.
   This conserves space in the main flash for storing code or other data.
-  Currently, you cannot combine storing P-GPS data in external flash with full modem FOTA.
```

Comment:
Have we written it anywhere that users need to enable FMFU dedicated partitions if they want to combine FMFU with P-GPS on external flash?

Reply:
🤔 probably not.  maybe it would be best to update this to something like, `if using PGPS with external flash and full modem fota, ensure the FMFU partition is used`

Reply:
> if using PGPS with external flash and full modem fota, ensure the FMFU partition is used

Yeah, I think adding that here would be an excellent idea!